docs/csharp/programming-guide/exceptions/codesnippet/CSharp/using-exceptions_3.cs

```diff
@@ -1,27 +1,32 @@
-        static void TestCatch2()
+        using System;
+        using System.IO;
+
+        public class ExceptionExample
```

Comment:
Won't it look odd that most examples in the article will be snippets with a `TestThrow`/`TestCatch`/etc. method, while this example will be a full program with `ExceptionExample` class?

Reply:
I think it does look odd, @svick. In fact, I find the code snippets somewhat odd -- without an entry point and invocation, they're not executable.  For new users (who are most likely to need to read this topic), I think that's confusing. I'll create an issue to standardize the code in this topic; what I really wanted to do here, though, was correct the exception handling.